docs/advanced-topics/wes/wes-agc-tutorial.rst

```diff
@@ -0,0 +1,332 @@
+
+    The AGC context and Dockstore configuration file do not need to be modified for the remainder of these examples, and will continue to function until the resources are modified and/or destroyed.
+
+Hello World Workflow
+---------------------
+The Dockstore entry associated with this workflow can be found here `agc-hello-world <https://dockstore.org/workflows/github.com/dockstore-testing/wes-testing/agc-hello-world:main?tab=info>`_.
+
+This WDL workflow prints out the string "Hello from AGC" as its output.
+
+    *Dockstore.wdl*
+
+    .. code:: text
+
+            version 1.0
+            workflow w {
+                call hello {}
+            }
+            task hello {
+                command { echo "Hello from AGC" }
+                runtime {
+                    docker: "ubuntu:latest"
+                }
+                output { String out = read_string( stdout() ) }
+            }
+
+1. Since this workflow is publicly posted on `Dockstore.org <https://dockstore.org/>`_, we can quickly launch it by passing the Dockstore CLI the entry name:
+
+    .. code:: text
+
+        dockstore workflow wes launch --entry github.com/dockstore-testing/wes-testing/agc-hello-world:main
+
+2. The above command will return a unique run ID, similar to:
+
+    .. code:: text
+
+        8e8e9f4b-fb1a-41df-bc37-9396d6f97db5
+
+    Copy the run ID and run the following to get the workflow run logs:
+
+    .. code:: text
+
+        dockstore workflow wes logs --id 8e8e9f4b-fb1a-41df-bc37-9396d6f97db5
+
+    The logs returned will look similar to:
+
+        .. code:: text
+
+            {
+              "run_id" : "8e8e9f4b-fb1a-41df-bc37-9396d6f97db5",
+              "request" : {
+                "workflow_params" : { },
+                "workflow_type" : "WDL",
+                "workflow_type_version" : "1.0",
+                "tags" : null,
+                "workflow_engine_parameters" : null,
+                "workflow_url" : null
+              },
+              "state" : "COMPLETE",
+              "run_log" : null,
+              "task_logs" : [ {
+                "name" : "w.hello|e6ce6c0a-ae99-43de-accc-4e43183de73f",
+                "cmd" : [ "echo \"Hello from AGC\"" ],
+                "start_time" : "2022-03-04T17:19:52.341Z",
+                "end_time" : "2022-03-04T17:23:17.196Z",
+                "stdout" : "s3://agc-example123-us-west-2/project/dockstoreAgcTutorialProject/userid/userM2QLG/context/ctx1/cromwell-execution/w/8e8e9f4b-fb1a-41df-bc37-9396d6f97db5/call-hello/hello-stdout.log",
+                "stderr" : "s3://agc-example123-us-west-2/project/dockstoreAgcTutorialProject/userid/userM2QLG/context/ctx1/cromwell-execution/w/8e8e9f4b-fb1a-41df-bc37-9396d6f97db5/call-hello/hello-stderr.log",
+                "exit_code" : 0
+              } ],
+              "outputs" : {
+                "id" : "8e8e9f4b-fb1a-41df-bc37-9396d6f97db5",
+                "outputs" : {
+                  "w.hello.out" : "Hello from AGC"
```

Comment:
Just curious, how do they handle massive output? Is it still all inline like this?

Reply:
Not sure, I'd expect all string outputs are inlined, but that hasn't been tested.